testdata/excel2json/expected_json_header.json

```diff
@@ -0,0 +1,76 @@
+{
+  "prefixes": {
+    "foaf": "http://xmlns.com/foaf/0.1/",
+    "dcterms": "http://purl.org/dc/terms/",
+    "schema": "https://schema.org/",
+    "sdh": "https://ontome.net/ns/sdhss/",
+    "cidoc": "http://www.cidoc-crm.org/cidoc-crm/"
+  },
+  "$schema": "https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/project.json",
+  "project": {
+    "shortcode": "11",
```

Comment:
A shortcode must always be 4 digits long, so I don't think it's a good idea to introduce wrong test data here.

Reply:
Great that you noticed it! I just noticed, that the read function does not specify a datatype, which means it is interpreted as an int and the leading zeros are removed. This could have been a stupid bug later!